src/core/a-animation.js

```diff
@@ -363,6 +364,8 @@ function getAnimationValues (el, attribute, dataFrom, dataTo, currentValue) {
     getForCoordinateComponent();
   } else if (['true', 'false'].indexOf(dataTo) !== -1) {
     getForBoolean();
+  } else if (dataTo && dataTo[0] === '#') {
```

Comment:
@dmarcos @ngokevin.  Alright on the drive home I was thinking the catching else statement does an automatic type conversion to number which will turn out to be a NaN property on the object.  So instead of checking prop or color I figured we can just check for `isNaN(dataTo)` and then feed it directly to the Three.js conversions.  This will always get a color (white) if an invalid color is passed and will still throw a warning from three.js when the color is not valid.

This gets around checking specific color formats and allows Three.js to handle improper input. Is there any reason you would think this would fail?